packages/base-plugin/lib/plugin.js

```diff
@@ -45,9 +45,10 @@ class BasePlugin {
    * @param {Record<string,unknown>} [cliArgs]
    */
   constructor(name, cliArgs = {}) {
+    super(`Plugin [${name}]`);
```

Comment:
I would prefer if plugin logger could also somehow share the the driver hash prefix, so it would be visible in logs which driver has initiated an action.

Also, usually people already have the `Plugin` suffix in their names, so I would just put the class name instead: `super(this.constructor.name)`